localstripe/resources.py

```diff
@@ -3176,6 +3206,57 @@ def _calculate_amount_in_tier(self, quantity, index):
         t = self.plan.tiers[index]
         return int(t['unit_amount']) * quantity + int(t['flat_amount'])
 
+    @classmethod
+    def _api_list_all(cls, url, subscription=None, limit=None,
+                      starting_after=None):
+        try:
+            if subscription is not None:
+                assert type(subscription) is str and subscription.startswith('sub_')
+
+        except AssertionError:
+            print('Invalid subscription provided')
+            raise UserError(400, 'Bad request')
+
+        li = super(SubscriptionItem,
+                   cls)._api_list_all(url, limit=limit,
+                                      starting_after=starting_after)
+
+        if subscription is not None:
+            li._list = [subitem for subitem in li._list if subitem._subscription == subscription]
+
+        return li
+
+    def _update(self, price_data=None, quantity=1, proration_behavior=None,
```

Comment:
Let's add a comment saying it's partially supported, since the real stripe api also accepts parameters such as `metadata` and `payment_behavior`?